src/components/action-popover/action-popover.spec.js

```diff
@@ -253,16 +256,69 @@ describe("ActionPopover", () => {
     }).not.toThrow();
   });
 
-  it('opens with the menu left position assigned so it is to right of the button if the "rightAlignMenu" prop is true', () => {
-    render({ rightAlignMenu: true });
-    const { menu } = getElements();
-    assertStyleMatch(
-      {
-        right: undefined,
-        left: "0",
-      },
-      menu
-    );
+  describe("it shoud render correct popper props", () => {
+    let menu;
+    let button;
+
+    it("if prop `top` is passed", () => {
```

Comment:
You've repeated this block a few times between here and line 308. Could the same outcome be achieved using a `it.each` instead? If you've done it this way for readability purposes, that's fine too.